src/v0.8/types/DatasetType.sol

```diff
@@ -98,5 +98,7 @@ library DatasetType {
         uint16 verificationsCount;
         mapping(address => Verification) verifications; // Address of the auditor who submits challenges.
         address[] auditors; // Records of auditors submitting verifications.
+        /// @dev This empty reserved space is put in place to allow future versions to add new
+        uint256[32] __gap;
```

Comment:
Including a reserved __gap in custom structures may increase gas fees; it's advisable to remove it